feature: define Staged Configs (#264)
nginx/documentation: content/nginx-one/changelog.md

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

## March 11, 2025

### Set up Staged Configurations
<!-- Set up include, with Glossary -->

> It allows you to save "work in progress." You can create it from scratch, an Instance, another Staged Config, or a Config Sync Group. It does _not_ have to be a working configuration until you publish it to an instance or a Config Sync Group. You can even maanage your **Staged Configurations** through our [API](link to staged configs API page). 

Reviewer: ```suggestion
It allows you to save "work in progress." You can create it from scratch, an Instance, another Staged Config, or a Config Sync Group. It does _not_ have to be a working configuration until you publish it to an instance or a Config Sync Group. You can even manage your **Staged Configurations** through our [API](link to staged configs API page). 
```

Reviewer 2: Could this be in bullets?

```
- Save Your Progress: Staged Configurations allow you to work on configuration changes without the need for a fully functional setup. You can create these drafts from scratch, an existing instance, another Staged Config, or a Config Sync Group.
- No Immediate Validation Required: There's no need to worry about immediate syntax or configuration issues. Your Staged Configuration doesn't have to be valid until you publish it to an instance or a Config Sync Group.
- Manage via API: You can easily manage your Staged Configurations programmatically through our [API](link to staged configs API page).
```

Author: ```suggestion
It allows you to:

- Save Your Progress: Staged Configurations allow you to work on configuration changes without the need for a fully functional setup. You can create these drafts from scratch, an existing Instance, another Staged Configuration, or a Config Sync Group.
- No Immediate Validation Required: You don't have to immediately address syntax or configuration issues. Your Staged Configuration doesn't have to be valid until you publish it to an Instance or a Config Sync Group.
- Manage through our API: You can easily manage your Staged Configurations programmatically through our [API]({{< ref "/nginx-one/api/api-reference-guide/#tag/StagedConfigs" >}}).

```

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

## March 11, 2025

> ### Set up Staged Configurations
> It allows you to:

Reviewer: ```suggestion
### Set up Staged Configurations

It allows you to:
```

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

## March 11, 2025

### Set up Staged Configurations

> <!-- Set up include, with Glossary -->

Author: ```suggestion
```

It allows you to:

- Save Your Progress: Staged Configurations allow you to work on configuration changes without the need for a fully functional setup. You can create these drafts from scratch, an existing Instance, another Staged Configuration, or a Config Sync Group.
- No Immediate Validation Required: You don't have to immediately address syntax or configuration issues. Your Staged Configuration doesn't have to be valid until you publish it to an Instance or a Config Sync Group.
- Manage through our API: You can easily manage your Staged Configurations programmatically through our [API]({{< ref "/nginx-one/api/api-reference-guide/#tag/StagedConfigs" >}}).

> 

Author: ```suggestion
```
